Chinese/ZZB/zzb.md

```diff
@@ -1,3 +1,6 @@
 至尊宝：谁说我斗鸡眼？我只是把视力集中在一点，以改变我以往对事物的看法！
 
-If you speak English, [go here](../../english/marshmallow.md).
```

Comment:
I don't know Chinese, so if you think that this line was out of place and should be removed, I will merge the pull request as long as you fix the broken link.